configure.ac

```diff
@@ -188,6 +188,17 @@ AC_SUBST(BLUEZ_CFLAGS)
 AC_SUBST(BLUEZ_LIBS)
 AM_CONDITIONAL(BLUETOOTH, test "${enable_bluetooth}" != "no")
 
+AC_ARG_ENABLE(cares, AC_HELP_STRING([--disable-cares],
+				[disable DNS client using c-ares library]),
+						[enable_cares=${enableval}])
+if (test "${enable_cares}" != "no"); then
+	PKG_CHECK_MODULES(CARES, libcares, dummy=yes,
```

Comment:
The actual library is called libc-ares2.  I actually tried applying as a patch to the ofono bzr tree, and the build failed during override_dh_auto_configure.  I'd added libc-ares-dev to the control file build dependencies, and verified that it was installed in my sbuild chroot, so something's still not right.  Note, I also added --enable_cares to the debian/rules file.  Can you please try a Debian build and report the changes you made?

I would also suggest that you use C_ARES as the Makefile variable, and change the "enable_cares" to "enable-c-ares" as most configure parameters used dashes as opposed to underscores.